### DISCUSSION TITLE
Putting datasets into the package

### DISCUSSION THREAD

**Discussion by Nambunga** [IBAHCM/RPiR#280] [Archive] [answered]
@richardreeve @soniamitchell Hello! again. I followed your instruction Sonia on how to commit large dataset and I made it successfully. But I started getting the error when putting the dataset into the package.
I attached the screenshots of error message and the view of both my data-raw and R folders for clarification to help me. Thanks :)
![IMG_4998](https://user-images.githubusercontent.com/77630539/107704144-b31f0600-6cb4-11eb-93d1-5dde6b845dea.JPG)
![IMG_4999](https://user-images.githubusercontent.com/77630539/107704155-b74b2380-6cb4-11eb-9099-47d97ec42653.JPG)
![IMG_5001](https://user-images.githubusercontent.com/77630539/107704162-ba461400-6cb4-11eb-987e-66d0789a2c8a.jpeg)

**Comment by soniamitchell** [comment, marked as answer]
Hey @Nambunga, `read.delim(file.path("ViewFullTable", "ViewFullTable.txt"))` is expecting `ViewFullTable.txt` to be in a directory called `ViewFullTable`. So you either need to create that directory, or change it to `read.delim("ViewFullTable.txt")` 👍

**Reply by richardreeve** [reply]
This is now also explained in the 4-1 practical text.
